File: src/kyu_7/parts_of_a_list/js/solution.js
function partlist(arr) {
    var newArray;
    var returnArray=[];
    for( var i=1; i<arr.length; i++){
        newArray = [];
        newArray.push(arr.slice(0,i).join(" "));
        newArray.push(arr.slice(i).join(" "));
        returnArray.push(newArray);
    }

    return returnArray;
}

module.exports = partlist;

File: src/kyu_7/parts_of_a_list/csharp/solution.cs
using System.Linq;
public class PartList
{
    public static string[][] Partlist(string[] arr)
    {
        return Enumerable.Range(1,arr.Length-1)
                         .Select(x=>
                             new string[]{
                                 string.Join(" ",arr.Take(x)),
                                 string.Join(" ",arr.Skip(x).Take(arr.Length-x))
                               })
                         .ToArray();

    }
}